Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLawSynthetics.xml

```diff
@@ -0,0 +1,19 @@
+﻿<Document>
+  # Synthetics
+  Synthetics must be given a chance to stand down, however must be repaired to serve its sentence if lethal force is used.
+
+  ## UNMC Synthetics
+  - UNMC Synthetics are programmed against being able to break marine law, and should not be arrested unless [color=#ffa500][bold]Provost Marshal Office[/bold][/color] have approved their arrest.
+  - These synthetic units are also granted the same rights and privileges of regular marines.
+  - If any [color=#ff000d][bold]MP[/bold][/color] believes they have broken marine law, a report should be faxed to the [color=#ffa500][bold]Provost Marshal Office[/bold][/color].
+  - If an officer orders the arrest of a Synthetic, the order may be ignored and the officer should be informed they must submit a report to allow the arrest.
```

Comment:
Rewrite suggestion below, note the use of the word must instead of may.

If an officer orders the arrest of a Synthetic which was not approved by the [color=#ffa500][bold]Provost Marshal Office[/bold][/color], [color=#ff000d][bold]MP[/bold][/color] must ignore the order and inform the officer.